sources/Makefile

```diff
@@ -102,15 +115,22 @@ postgres:
 	@echo "+\n++ Make: Run local db.\n+"
 	@docker-compose -f docker-compose.yml up postgres
 
+# Local Queue
+redis:
+	@echo "+\n++ Make: Run local queue.\n+"
+	@docker-compose -f docker-compose.yml up redis
+
 # Local API only
 api-only:
 	@echo "+\n++ Make: Building and running API only"
 	@docker-compose -f docker-compose.yml up -d postgres
+	@docker-compose -f docker-compose.yml up -d redis
```

Comment:
Can we add this as a dependency in the docker-compose for API/Workers/Queues?
Same for the others.